examples/plot_wavelet.py

```diff
@@ -67,3 +67,50 @@
 axs[1, 1].set_title("DWT2 coefficients (zeroed)")
 axs[1, 1].axis("tight")
 plt.tight_layout()
+
+###############################################################################
```

Comment:
It may be good to also modify the top description of this example from

```
This example shows how to use the :py:class:`pylops.DWT` and
:py:class:`pylops.DWT2D` operators to perform 1- and 2-dimensional DWT.
```

to 

```
This example shows how to use the :py:class:`pylops.DWT` and
:py:class:`pylops.DWT2D` and :py:class:`pylops.DWTND` operators to perform 1-, 2-, and N-dimensional DWT.
```

Reply:
Good point! I made the suggested change.